src/appengine/handlers/cron/cleanup.py

```diff
@@ -948,16 +1005,16 @@ def update_issue_ccs_from_owners_file(testcase, issue):
   issue_comment = (
       'Automatically adding ccs based on OWNERS file / target commit history.')
   if utils.is_oss_fuzz():
-    issue_comment += OSS_FUZZ_INCORRECT_COMMENT
+    issue_comment += OSS_FUZZ_INCORRECT_COMMENT + '.'
   else:
-    issue_comment += INTERNAL_INCORRECT_COMMENT
-  issue_comment += '.'
+    issue_comment = _append_generic_incorrect_comment(issue_comment, policy,
```

Comment:
should this be `+=` ? If not, maybe add a comment, because `_append...` function name creates a little confusion.

Reply:
hmm, I think += is even more confusing, because the _append in the name is not actually appending anything. 

This is similar to e.g. string.replace(), like a = a.replace('123', 'foo', 'bar'). In Go there is a similar thing e.g.: `a = append(a, 5)` so.